src/js/views/eva-variant-view.js

```diff
@@ -156,143 +524,222 @@ EvaVariantView.prototype = {
-                    ensemblTranscriptId = annotationDetails.ensemblTranscriptId;
-                }
-                _consequenceTypeTable += '<tr><td class="variant-view-ensemblGeneId">' + ensemblGeneId + '</td><td class="variant-view-ensemblTranscriptId">' + ensemblTranscriptId + '</td><td class="variant-view-link">' + link + '</td><td class="variant-view-soname">' + this[key].soName + '&nbsp;' + svg + '</td></tr>'
-            }, soTerms);
-
-        }, annotation);
-        _consequenceTypeTable += '</tbody></table></div></div>'
-
-        return _consequenceTypeTable;
-
-    },
-    _renderConservedRegionData: function (data) {
-        var conservedRegionScores = data[0].annotation.conservedRegionScores;
-        if (!conservedRegionScores) {
-            return '';
-        }
-        var _conservedRegionTable = '<div class="row"><div class="col-md-8"><table class="table hover">'
-        _conservedRegionTable += '<tr><th>Source</th><th>Score</th></tr>'
-        _.each(_.keys(conservedRegionScores), function (key) {
-            console.log(this[key])
-            _conservedRegionTable += '<tr><td>' + this[key].source + '</td><td>' + this[key].score + '</td></tr>'
-
-        }, conservedRegionScores);
-        _conservedRegionTable += '</table></div></div>'
-
-        return _conservedRegionTable;
+    _renderConsequenceTypeData: function (variantDataArray) {
+        var _this = this;
+        var variantIndex = 1;
+        return variantDataArray.map(function(data) {
+            var consequenceTypeHeading = '<h4 class="variant-view-h4"> Consequence Types' + (data.repr ? " for "+data.repr : "") +  '</h4>';
+            var noDataAvailableSection = consequenceTypeHeading + '<div>No Consequence Type data available</div>';
+            if(_.isUndefined(data.annotation)){
+              return noDataAvailableSection;
+            }
+            var annotation = data.annotation.consequenceTypes;
+            if (!annotation) {
+                return noDataAvailableSection;
+            }
+            annotation = annotation.sort(_this._sortBy('ensemblGeneId', _this._sortBy('ensemblTranscriptId')));
+            var _consequenceTypeTable = consequenceTypeHeading + '<div class="row"><div><table id="consequence-type-summary-' + variantIndex + '" class="table hover" style="font-size: small">';
+            _consequenceTypeTable += '<thead><tr><th>Ensembl Gene ID</th><th>Ensembl Transcript ID</th><th>Accession</th><th>Name</th></tr></thead><tbody>';
+            _.each(_.keys(annotation), function (key) {
+                var annotationDetails = this[key];
+                var soTerms = this[key].soTerms;
+                _.each(_.keys(soTerms), function (key) {
+                    var link = '<a href="http://www.sequenceontology.org/miso/current_svn/term/' + this[key].soAccession + '" target="_blank">' + this[key].soAccession + '</a>';
```

Comment:
I think this URL is declared as a constant somewhere else.

Reply:
The one that is declared as constant is SO_SERVICE = "http://www.sequenceontology.org/browser/current_release/term" which is different from this one. Anyway, I will refactor this into a global variable too.

Reply:
alright. I misunderstood this. I thought this was invoking a different service but it I am not sure why we use "Current SVN" as opposed to the "Current Release" here. Should we keep it as it is or change this to use "Current Release" too?

Reply:
Let's use current release, because that is the stable version.